templates/admin/list/list_youtube_id.html.twig

```diff
@@ -0,0 +1,9 @@
+{% extends get_admin_template('base_list_field', admin.code) %}
+
+{% block field %}
+    <div style="text-align: center;">
```

Comment:
This Sonata Admin includes Bootstrap 3, so you can apply this class here : 

```html
<div class="text-center">
```

Reply:
Thanks CSS sensei

Reply:
:man_cartwheeling: